src/components/IDO/IDO.tsx

```diff
@@ -0,0 +1,124 @@
+  const getButtonMessage = () => {
+    return 'Connect a wallet'
+  }
+
+  const [tokenFromIndex, setTokenFromIndex] = React.useState<number | null>(
+    tokens.length ? 0 : null
+  )
+  return (
+    <Grid container className={classes.containerIDO}>
+      <Grid sm={12} item className={classes.header}>
+        <Typography component='h1'>IDO</Typography>
+      </Grid>
+      <Grid item sm={7} className={classes.IDOWrapper}>
+        <Box>
+          <Grid className={classes.root}>
+            <Grid className={classes.header}>
+              <Typography component='h2'>Deposit your SOL</Typography>
+            </Grid>
+            <Box className={classes.tokenComponentTextContainer}>
+              <Typography className={classes.tokenComponentText}>Est.: 56.0278$</Typography>
+              <Typography className={classes.tokenComponentText}>Balance: 1004.5 SOL </Typography>
+            </Box>
+            <ExchangeAmountInput
+              setValue={() => {}}
+              decimal={6}
+              placeholder={'0.0'}
+              onMaxClick={() => {}}
+              current={tokenFromIndex !== null ? tokens[tokenFromIndex] : null}
+              tokens={tokens}
+              onSelect={(name: string) => {
+                setTokenFromIndex(
+                  tokens.findIndex(token => {
+                    return name === token.symbol
+                  })
+                )
+              }}
+            />
+            <Grid>
+              <Typography className={classes.tokenComponentText}>Deposited amount:</Typography>
+            </Grid>
+            <Box className={classes.valueBox}>
+              <CardMedia image={invariantLogo} className={classes.iconLogo} />
+              <Box className={classes.infoContainer}>
+                <Box>
+                  <Typography className={classes.depositAmouted}>46.643 xUSD</Typography>
+                </Box>
+                <Box className={classes.boxInfo}>
+                  <Typography className={classes.valueInfo}>47.43 USD</Typography>
+                  <Typography className={classes.valueInfo}> 0.0432 SOL</Typography>
+                  <Typography className={classes.valueInfo}> 0.0000 xETH</Typography>
+                  <Typography className={classes.valueInfo}>0.0000 xBTC</Typography>
+                </Box>
+              </Box>
+            </Box>
+            <OutlinedButton
+              name={getButtonMessage()}
+              className={classes.buttonBuy}
+              color='primary'
+              onClick={() => {}}
+            />
+          </Grid>
+        </Box>
+      </Grid>
+      <Grid item sm={4} className={classes.info}>
+        <Box className={classes.containerInfo}>
+          <Typography className={classes.labelInfo}>Sale period ends in</Typography>
+          <Box className={classes.wrapperInfo}>
+            <CardMedia image={watchIcon} className={classes.icon} />
+            <Typography className={classes.textInfo}>15:30:33</Typography>
```

Comment:
i think timers shouldn't be static, you can use animated numbers, we use them in some components